analytics-integrations/taplytics/build.gradle

```diff
@@ -0,0 +1,5 @@
+apply from: rootProject.file('gradle/integration.gradle')
+
+dependencies {
+  compile files('libs/taplytics.jar')
```

Comment:
No maven dependency, It's done like this:

https://github.com/taplytics/Taplytics-Android-SDK#android-studio-installation

I'll get an artifactory up one day though.